project-management/12_risks/risk_register.md

```
@@ -1,161 +1,239 @@
+## Risk Summary Dashboard
+
+| Risk Level | Count | Percentage |
+|------------|-------|------------|
+| High | 2 | 22% |
+| Medium | 5 | 56% |
+| Low | 2 | 22% |
+| **Total** | **9** | **100%** |
+
+## Risk Matrix
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)